sunpy/visualization/animator/base.py

```diff
@@ -558,7 +558,7 @@ def plot_start_image(self):  # pragma: no cover
         raise NotImplementedError("Must be defined and used by subclasses of {0}.".format(self.__class__))
 
     @abc.abstractmethod
-    def update_plot(self, val, slider):  # pragma: no cover
+    def update_plot(self, val, im, slider):
```

Comment:
it feel weird calling this `im` here, as it is either `im` or `line` or some other plot object. Maybe `artist` is a good name?

Reply:
Sure, it's better. I'll update this.